docker/Dockerfile

```diff
@@ -158,12 +158,8 @@ RUN apt-get update && \
 ARG NETWORK
 ENV NETWORK ${NETWORK:-Mainnet}
 
-# Set this to enable the RPC port
-ARG RPC_PORT
-ENV RPC_PORT ${RPC_PORT}
-
 # Expose configured ports
-EXPOSE 8233 18233 $RPC_PORT
+EXPOSE 8233 18233
```

Comment:
Zebra uses only one of these ports, depending on whether it runs on Mainnet or Testnet. Is it OK to have an extra port here?

Also, what about the rest of the ports?

Reply:
As far as I'm aware, the _default_ zebrad configuration just opens one of these ports `8233` or `18233`, based on the chosen Network. One of these ports is expected to _always_ be exposed.

If there's any other port that's expected to opened by default, it should be added here.